documentation/modules/auxiliary/admin/dcerpc/cve_2021_1675_printnightmare.md

```diff
@@ -66,6 +66,9 @@ due to how these payloads are synchronized. If the session closes, dies, or the
 DLL can be used again otherwise a new DLL will need to be created for each simultaneously active session. One DLL can
 however be used to establish multiple sessions simultaneously so long as each session is on a different target.
 
+If this value is a UNC path, it will automatically be converted to use the `\??\UNC\` prefix instead of `\\` to bypass
+the path check in certain scenarios.
+
 ### ReconnectDelay
 *This is an advanced option.*
 
```

Comment:
The documentation will need to be updated given the new changes such as deregistering the `AutoCheck` option.

Reply:
AutoCheck isn't a documented option here so I'm not sure what needs to be updated.